lib/gplt.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "thor"
+require "./tests/helpers/perf_run_helper"
+
+# Add pry support when available
+begin
+  require "pry"
+rescue LoadError # rubocop: disable Lint/HandleExceptions
+  # do nothing
+end
+
+module GPLT
+  class Error < StandardError; end
+
+  # Class to define CLI subcommands
+  class CLI < Thor
+    extend PerfRunHelper
+
+    default_test_type = "apples to apples"
+
+    option :test_type, default: default_test_type
```

Comment:
All of the other standalone scripts for GPLT feature short versions of the options. It would be nice if this followed that pattern.

They also feature a `-h` option to display the help. It seems weird to have a totally different usage pattern for this script:
```
be bin/gplt help validate2baseline
```
vs. (for examlple)
```
be ruby util/abs/provision_pe_xl_nodes.rb -h
```

Reply:
I am a big fan of -h for help as well...   It looks like the top level with no subcommand kinda prints out the help if you do -h, but only because that is the default for anything it doesn't understand... so that one is fine.  But setting up 
`be bin/gplt validate2baseline -h`
To also give the help would be really nice.  Right now it give a pretty lame message 
`No value provided for required options '--baseline', '--results-dir'`
Which doesn't even tell you how to get the help you are looking for.